### PR TITLE
update chronometer.sh

### PR DESCRIPTION
**Pull request by tuplink**
_This template was created based on the work of [`udemy-dl`](https://github.com/nishad/udemy-dl/blob/master/LICENSE)._

**By submitting this pull request, I confirm the following (please check boxes, eg [X]):**

- [x ] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md).
- [x ] I have checked that [another pull request](https://github.com/pi-hole/pi-hole/pulls) for this purpose does not exist.
- [x ] I have considered, and confirmed that this submission will be valuable to others.
- [x ] I accept that this submission may not be used, and the pull request closed at the will of the maintainer.
- [x ] I give this submission freely, and claim no ownership to its content.

**How familiar are you with the codebase?:**

- [ ] 1 (very unfamiliar)
- [ ] 2
- [ ] 3
- [ ] 4
- [x ] 5
- [ ] 6
- [ ] 7
- [ ] 8
- [ ] 9
- [ ] 10 (very familiar)

not sure how much use this will get but
updated as piholeIPv6file="/etc/pihole/.useIPv6" is no longer used

### CODE REVIEW

**Comment by PromoFaux** [comment]
I think I'd like to take this script in a different direction. Take a look at it's [implementation in the `db` branch](https://github.com/pi-hole/pi-hole/blob/db/advanced/Scripts/chronometer.sh) (which is still going to need some work)

**Comment by tuplink** [comment]
have a look at https://raw.githubusercontent.com/tuplink/pi-hole/3b4bdaf052b4b362fef046b6c570cd4cfb0ef33a/advanced/Scripts/chronometer.sh and see if thats more of what you are looking for 

python becomes a dependency    if this is the route you want to go let me know and ill wait for a bunch of these changes to go through  so that  i can update basic-install.sh and pihole

File: automated install/basic-install.sh [hunk 1]
```diff
@@ -791,7 +791,7 @@ finalExports() {
     fi
     {
     echo "piholeInterface=${piholeInterface}"
-    echo "IPv4_address=${IPv4_address}"
+    echo "IPv4_address=${IPv4_address%/*}"
```
Comment (tuplink) [inline automated install/basic-install.sh]: so yea i was testing it

**Comment by dschaper** [comment]
Can you rebase your changes off of the development branch? That's going to be your best bet for a target, as other branches are unstable. (Note the new merge conflicts...)

**Pull request closed without merging**